terraform/scripts/modify_variable.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+#script logging to modify_variable.log file
+test x$1 = x$'\x00' && shift || { set -o pipefail ; ( exec 2>&1 ; $0 $'\x00' "$@" ) | tee -a modify_variable.log ; exit $? ; }
+
+vm_floating_ip_cidr="193.190.80.0/25"
+vsc_floating_ip_cidr="172.24.48.0/20"
```

Comment:
Users probably do not have to change this but maybe we should move it into `terraform/scripts/modify_variable.config`

Reply:
moved to modify_variable.config by bf06b4c8d2ee6854b9ff6fe6d50fa8511fac60b2